spec/unit/services/cookie_service_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+
+describe Locomotive::Steam::CookieService do
+
+  let(:steam_cookies)   { {} }
+  let(:request_cookies) { {} }
+  let(:request)         { instance_double('Request', env: { 'steam.cookies' => steam_cookies }, cookies: request_cookies) }
+  let(:cookie)          { {'value' => 'bar2'} }
+  let(:service)   { described_class.new(request)}
```

Comment:
indentation :-)

Reply:
what do you think about using rubocop + pre-commit with auto formating of the code and checking it localy before commit?
https://github.com/rubocop-hq/rubocop
https://pre-commit.com/
On python side we start to use same stuff (black that format all the code)